src/common-lib/urls/getDeploymentTestUrls.js

```diff
@@ -71,6 +71,11 @@ function getDeploymentTestUrls() {
     "/teachers/curriculum/english-secondary-aqa/units",
     "/teachers/curriculum/english-secondary-aqa/overview",
     "/teachers/curriculum/english-secondary-aqa/downloads",
+    "/teachers/curriculum/science-secondary-aqa/units",
+    "/teachers/curriculum/science-secondary-aqa/overview",
```

Comment:
I think this is redundant because it's not significantly different to the English one. But we could add in a cycle 2 explainer page when that goes live?